XRTK.SDK/Packages/com.xrtk.sdk/Features/UX/Scripts/Collections/BaseObjectCollection.cs

```diff
@@ -18,7 +18,7 @@ public abstract class BaseObjectCollection : MonoBehaviour
         /// <summary>
         /// List of objects with generated data on the object.
         /// </summary>
-        protected List<ObjectCollectionNode> NodeList { get; } = new List<ObjectCollectionNode>();
+        public List<ObjectCollectionNode> NodeList { get; } = new List<ObjectCollectionNode>();
```

Comment:
Should return a `IReadonlyLIst<ObjectCollectionNode>`

Might wanna pull this out into it's own pr with the undo ports as well.